examples/chart/teleport/values.yaml

```diff
@@ -118,6 +118,11 @@ proxy:
     # We assume TLS is terminated in front of the proxy by default
     enabled: true
 
+license:
+  enabled: true
```

Comment:
A comment here that says "set false to run Teleport in Community edition would be helpful"

Reply:
sure, thanks